modules/tinymce/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - When deleting the last row in a table, the cursor would jump to the first cell (top left), instead of moving to the next adjacent cell in some cases. #TINY-6309
+- The function `schema.isWrapper` doesn't exclude the elements that are not eligible as a wrapper. #TINY-10385
```

Comment:
Maybe we should also mention the `schema.isInline` predicate as well.

Reply:
changed